libcxx/utils/generate_feature_test_macro_components.py

```diff
@@ -2060,12 +2109,106 @@ def implemented_ftms(self) -> Dict[str, Dict[str, Any]]:
 
         return get_ftms(self.__data, self.std_dialects, True)
 
+    @functools.cached_property
+    def ftm_meta_data(self) -> Dict[str, Dict[str, Any]]:
```

Comment:
```suggestion
    def ftm_metadata(self) -> Dict[str, Dict[str, Any]]:
```

Metadata is generally spelled as a single word, I think?